container-make.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+
+set -x
+
+CONTAINER_NAME=${1}
+CONTAINER_TAG=${2}
+
+PROJECT_NAME='github.com/tracer/tracer'
+PROJECT_DIR="${PWD}"
+
+CONTAINER_GOPATH='/go'
+CONTAINER_PROJECT_DIR="${CONTAINER_GOPATH}/src/${PROJECT_NAME}"
+CONTAINER_PROJECT_GOPATH="${CONTAINER_GOPATH}"
+
+docker run --rm \
+        -v ${PROJECT_DIR}:${CONTAINER_PROJECT_DIR} \
+        -e GOPATH=${CONTAINER_PROJECT_GOPATH} \
+        -e CGO_ENABLED=0 \
+        -e GODEBUG=netdns=go \
+        -w "${CONTAINER_PROJECT_DIR}" \
+        golang:1.7rc1-alpine \
+        go build -v -o tracer ${PROJECT_NAME}/cmd/tracer
+
+docker run --rm \
+        -v ${PROJECT_DIR}:${CONTAINER_PROJECT_DIR} \
+        -e GOPATH=${CONTAINER_PROJECT_GOPATH} \
+        -e CGO_ENABLED=0 \
+        -e GODEBUG=netdns=go \
+        -w "${CONTAINER_PROJECT_DIR}" \
+        golang:1.7rc1-alpine \
+        go build -v -o tracer-cli ${PROJECT_NAME}/cmd/tracer-cli
+
+strip "${PROJECT_DIR}/tracer"
+strip "${PROJECT_DIR}/tracer-cli"
```

Comment:
Actually, stripping Go binaries should work just fine nowadays. The last bug was years ago, affecting ARM only. In the early days, Go produced slightly wrong ELF output that confused strip, but that's been fixed.

However, I'd still advise against stripping in this specific case, as it makes debugging needlessly difficult.